README.rst

```diff
@@ -64,6 +64,9 @@ Note that the makefile makes sure the requirements.txt is updated with
 "pip-compile" when setup.py or requirements.in changes. You can also run it by
 hand.
 
+A requirements.txt file isn't really needed, as it is a library. It was added
+to help keep track of versions when updating.
```

Comment:
@caspervdw : ik heb even een regeltje toegevoegd over de requirements.txt. Hij is inderdaad niet nodig qua deployment. Maar ik vond 'em erg handig om in de gaten te kunnen houden welke versies er nou gebruikt werden tijdens het upgraden van de django versies :-)